tools/nnyeah/tests/integration/IntegrationExamples.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+using Xamarin;
+using Xamarin.Tests;
+using Xamarin.Utils;
+
+namespace Microsoft.MaciOS.Nnyeah.Tests.Integration {
+	[TestFixture]
+	public class IntegrationExamples {
+		string IntegrationRoot => Path.Combine (Configuration.SourceRoot, "tools", "nnyeah", "integration");
+		string Nnyeah => Path.Combine (Configuration.SourceRoot, "tools", "nnyeah", "nnyeah", "bin", "Debug", "net6.0", "nnyeah.dll");
+
+		// TODO - This code, and passing xamarin-assembly/microsoft-assembly should be removed long term from nnyeah
+		string GetLegacyPlatform (ApplePlatform platform)
+		{
+			switch (platform)
+			{
+				case ApplePlatform.MacOSX:
+					return Configuration.GetBaseLibrary (TargetFramework.Xamarin_Mac_2_0_Mobile);
+				case ApplePlatform.iOS:
+					return Configuration.GetBaseLibrary (TargetFramework.Xamarin_iOS_1_0);
+				default:
+					throw new NotImplementedException ();
+			}
+		}
+
+		string GetNETPlatform (ApplePlatform platform)
```

Comment:
```suggestion
		string GetNetPlatform (ApplePlatform platform)
```
Perhaps this may be better style?

Reply:
I'm going to do this in the next PR, just to save CI time.